web-registry/src/components/molecules/ProjectSelect.tsx

```diff
@@ -25,47 +26,46 @@ const ProjectSelect: React.FC<FieldProps> = ({
   saveOptions,
   ...props
 }) => {
-  const { data: dbDataByOnChainId } = useCreditClassByOnChainIdQuery({
-    variables: { onChainId: creditClassId as string },
-  });
   const { setFieldValue } = useFormikContext();
   const [projectOptions, setProjectOptions] = useState<Option[]>([]);
 
   useEffect(() => {
+    let ignore = false;
+
+    const fetchData = async () => {
+      if (!creditClassId) return;
+      const data = await queryProjectsByClass(creditClassId);
+
+      const options =
+        data?.projects.map((project: ProjectInfo) => {
+          return {
+            label:
+              `${startCase(project?.metadata?.['schema:name' as any] || '')} ${
+                project.id ? '(' + project.id + ')' : ''
```

Comment:
Should this be a template string ?
```suggestion
                project.id ? `(${project.id})` : ''
```